src/app/containers/ArticleMetadata/index.test.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { render, waitForDomChange } from '@testing-library/react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import ArticleMetadata from './index';
+import { articleDataNews, articleDataPersian } from '../Article/fixtureData';
+
+const getISOStringDate = date => new Date(date).toISOString();
+
+// eslint-disable-next-line react/prop-types
+const Context = ({ service, children }) => (
+  <ServiceContextProvider service={service}>
+    <RequestContextProvider
+      bbcOrigin="https://www.test.bbc.co.uk"
+      id="c0000000000o"
+      isAmp={false}
+      pageType="article"
+      pathname="/pathname"
+      service={service}
+      statusCode={200}
+    >
+      {children}
+    </RequestContextProvider>
+  </ServiceContextProvider>
+);
+
+const ArticleMetadataForNewsInternational = () => (
+  <Context service="news">
+    <ArticleMetadata
+      articleId={articleDataNews.metadata.id}
+      author="https://www.facebook.com/bbcnews"
+      firstPublished={getISOStringDate(articleDataNews.metadata.firstPublished)}
+      lastPublished={getISOStringDate(articleDataNews.metadata.lastPublished)}
+      section={articleDataNews.metadata.passport.genre}
+      aboutTags={articleDataNews.metadata.tags.about}
+      mentionsTags={articleDataNews.metadata.tags.mentions}
+      lang={articleDataNews.metadata.passport.language}
+      title={articleDataNews.promo.headlines.seoHeadline}
+      seoHeadline={articleDataNews.promo.headlines.seoHeadline}
+      description={articleDataNews.promo.headlines.seoHeadline}
+      linkedData={{
```

Comment:
Are these props old? I dont think this is used anymore?

Reply:
Oh I think you're right